sdk/repo/verify.go

```diff
@@ -162,6 +162,38 @@ func (r *repo) projectBranches(p Project) ([]string, error) {
 	return branches, nil
 }
 
+func (r *repo) projectTags(p Project) ([]string, error) {
+	git := exec.Command("git", "show", "-s", "--pretty=%D", "HEAD")
+	git.Dir = filepath.Join(r.root, p.Path)
+	git.Stderr = os.Stderr
+	out, err := git.Output()
+	if err != nil {
+		return nil, err
+	}
+
+	var tags []string
+
```

Comment:
Can you add sample output in a comment, to help understand what the code does?

Reply:
Sure